doc/source/serve/doc_code/monitoring/logging_config.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import logging
+from ray import serve
+
+#  __json_start__
```

Comment:
Please also make changes in https://docs.ray.io/en/latest/serve/configure-serve-deployment.html and https://docs.ray.io/en/latest/serve/production-guide/config.html# as relevant

Reply:
done